spec/features/5_photos_spec.rb

```diff
@@ -1,120 +1,126 @@
 require "rails_helper"
 
-show_tests_in_browser = false
-do_not_show_tests_in_browser = false
+feature "Photos:" do
+
+  context "photos show page" do
+    before(:each) do
+      @user_1 = FactoryGirl.create(:user, :username => "1", :email => "1@m.com")
+      @user_2 = FactoryGirl.create(:user, :username => "2", :email => "2@m.com")
```

Comment:
How about using [factories with sequences](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#sequences
) to clean this up? Since factories should be [minimally viable out of the box](https://robots.thoughtbot.com/factories-should-be-the-bare-minimum), we could just do something like

```ruby
owner = create(:user)
not_owner = create(:user)
```

(Note that avoiding `before`, `let`, and using factories this way allows for descriptive variable names and much less noise that **makes the intention of the tests clearer**.)